### ISSUE TITLE
Skin tone disability issue

### ISSUE THREAD

**Issue by suneelfm-ai** [ealush/emoji-picker#322]
**Describe the bug**
Even though if we disabled the skin tone. The library is allowing user to changes the skin tone by long press on emoji

**To Reproduce**
1. Disable the skin tone 
2. Long press on the emoji which supports the skin tone
3. Observe the skin tone selection popup

**Expected behavior**
- The skin tone selection should be disabled on long press on emoji also.
- The icon should not be displayed on disable the skin tone

**Screenshots**

![New Image (4)](https://user-images.githubusercontent.com/117177361/218392395-8bb15cd9-6296-44ac-ab45-e07887ab3bea.jpeg)

**Desktop (please complete the following information):**

- OS: [windows 10,(actually for all the OS)]
- Browser [Chrome (all the browsers)]
- Version [all version]

**Comment by ealush** [comment]
Yep. I think you're right. I can get to it in the weekend, but it should be fairly simple to implement if you want to try yourself before that.

**Comment by ealush** [comment]
Here's the relevant handler: https://github.com/ealush/emoji-picker-react/blob/master/src/hooks/useMouseDownHandlers.ts#L79
